tasks/main.yml

```diff
@@ -6,44 +6,7 @@
     cache_valid_time=3600
   with_items:
     - python-meld3
```

Comment:
Good point, the apt package will take care of its dependencies on its own. Gone e097bd4